nyc_taxis/README.md

```diff
@@ -60,8 +60,11 @@ This track allows to overwrite the following parameters with Rally 0.8.0+ using
 * `bulk_size` (default: 10000)
 * `bulk_indexing_clients` (default: 8): Number of clients that issue bulk indexing requests.
 * `ingest_percentage` (default: 100): A number between 0 and 100 that defines how much of the document corpus should be ingested.
+* `conflict_probability` (default: 25): A number between 0 and 100 that defines the probability of id conflicts. This requires to run the respective challenge.
```

Comment:
How about instead of "This requires to run the respective challenge." to simply and explicitly say "Only used by the `update` challenge".